instance/models/mixins/openedx_storage.py

```diff
@@ -61,7 +61,10 @@ def get_storage_settings(self):
         Only needed when not using ephemeral databases
         """
         if self.use_ephemeral_databases:
-            return ''
+            # Workaround for broken CMS course export/import
+            # caused by https://github.com/edx/edx-platform/pull/14552
+            template = loader.get_template('instance/ansible/ephemeral.yml')
+            return template.render()
```

Comment:
@replaceafill Can we make sure to revert once the upstream PR is merged? To ensure we don't forget, I would only close the current ticket once both the upstream PR is merged and this reverted.